personal-website/src/app/globals.css

```diff
@@ -0,0 +1,359 @@
+:root {
+  /* variables */
+  --cg-black: rgb(30, 32, 32);
+  --cg-blue: rgb(59, 179, 242);
+  --cg-grey: rgb(110, 126, 140); /* text color */
+  --cg-light-grey: rgb(119, 156, 192); /* highlight text color */
+  --cg-white: rgb(255, 255, 255);
+  --cg-dark-gunmetal: rgb(11, 22, 34); /* page background color */
+  --cg-light-gunmetal: rgb(21, 31, 46);
+  --cg-blue-gunmetal: rgb(21, 34, 50);
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  background: var(--cg-dark-gunmetal);
+  position: relative;
+  min-height: 100vh;
+  padding: 0;
+  margin: 0;
+  font-family: monospace;
+}
+
+/* CSS for navigation */
+
+.navbar {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  height: 7%;
+  z-index: 10;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+
+  background-color: var(--cg-blue-gunmetal);
+  color: var(--cg-white);
+  padding: 0px 0px;
+
+  /* Debug border */
+  /* border: solid red; */
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  right: 30px;
+  list-style: none;
+
+  /* Debug border */
+  /* border: solid blue; */
+}
+
+.navbar a {
+  color: var(--cg-grey);
+  font-weight: bold;
+  font-size: 24px;
+  margin: 0 20px;
+  text-decoration: none;
+  letter-spacing: 2px;
+
+  font-family: monospace;
+}
+
+.navbar a:hover {
```

Comment:
In most cases, it is better to create new css files that are dedicated to each page. This better modularizes the code